README.md

```diff
@@ -68,6 +68,19 @@ Kanban-style boards to manage work items. Integrates well with teams and other s
 | Platform  | macOS, Windows, iOS, Android, web              |
 | URL       | https://trello.com/                            |
 
+### Taiga
+
+Taiga is a project management tool for multi-functional agile teams. It has a rich feature set and at the same time it is very simple to start with through its intuitive user interface.
+
+![Taiga](media/tool-taiga.gif)
```

Comment:
Nit - could you please update this with a description of what is happening in the screenshot, since this is used for the alt text?